.github/workflows/build.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
-          java-version: 11
+          java-version: 17
```

Comment:
if we build the binary with JDK 17, then consumers on Java 11 wont' be able to use it?

Reply:
yes I think so, but this is a requirement for AGP 8 anyway right?

Reply:
The AGP JDK requirement is only for the Java running Gradle.
The toolchain for the Test task can still be lower, can't it?